viz.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<p>You could try the more OO API.</p><p> </p><pre class="ql-syntax" spellcheck="false">ax, f = plt.subplots()
arr = clr.matrix()[:]
im = ax.matshow(arr, vmax=500)
ax.set(
    xticks=(chromstarts, clr.chromnames),
    xlabel='position, chrom#',
    ylabel='position, bin#',
)

# I think you can use plt.colorbar or f.colorbar. I prefer plt
plt.colorbar(im, ...)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/examples/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>